python/Ganga/GPIDev/Lib/File/LocalFile.py

```diff
@@ -243,53 +261,54 @@ def put(self):
                 shutil.copy(path.join(job.outputdir, self.namePattern),
                             path.join(job.outputdir, self.localDir, self.namePattern))
            
-
     def cleanUpClient(self):
         """
         This performs the cleanup method on the client output workspace to remove temporary files
         """
         # For LocalFile this is where the file is stored so don't remove it
         pass
 
-## rcurrie Attempted to implement for 6.1.9 but commenting out due to not being able to correctly make use of setLocation
+    def getWNScriptDownloadCommand(self, indent):
+
+        # create symlink
+        shortScript = """
+# create symbolic links for LocalFiles
+for f in ###FILELIST###:
+    if not os.path.exists(os.path.basename(f)):
+        os.symlink(f, os.path.basename(f))
+"""
+        from Ganga.GPIDev.Lib.File import FileUtils
```

Comment:
And again, circular imports due to all of the extra mess in `__init__.py` reverting